src/test/java/bio/terra/service/dataset/SelfHostedDatasetIntegrationTest.java

```diff
@@ -138,8 +138,16 @@ public void testSelfHostedDatasetWithDedicatedSALifecycle() throws Exception {
     testSelfHostedDatasetLifecycle("jade_testbucket_no_jade_sa", true);
   }
 
+  @Test
+  public void testSelfHostedDatasetRequesterPaysLifecycle() throws Exception {
+    testSelfHostedDatasetLifecycle("jade_testbucket_requester_pays", false);
```

Comment:
Not following exactly, where does this bucket come from? We aren't creating it in the test, right?

Reply:
Ah, that's one that has to exist before we run. It lives in the `broad-jade-dev` project